src/ucs/sys/topo/base/topo.c

```diff
@@ -412,11 +392,11 @@ ucs_topo_get_distance_sysfs(ucs_sys_device_t device1,
         return UCS_OK;
     } else if (ucs_topo_is_sys_root(common_path)) {
         if (ucs_topo_is_same_numa_node(device1, device2)) {
-            ucs_topo_common_numa_node_distance(distance);
+            *distance = ucs_global_opts.dist_phb;
```

Comment:
If we are talking in terms of `nvidia-smi` tool, this distance was designed to represent not PHB but NODE distance.

Reply:
Changed to `node`.